extensions/wikia/AdEngine/js/spec/lookup/adapter/appnexusPlacements.spec.js

```diff
@@ -0,0 +1,69 @@
+describe('ext.wikia.adEngine.lookup.adapter.appnexusPlacements', function () {
+	'use strict';
+
+	var mocks = {
+		adLogicZoneParams: {
+			getVertical: function() {}
+		},
+		window: {
+			location: {
+				host: ''
+			}
+		}
+	}, testCases = [
+		{
+			vertical: 'lifestyle',
+			skin: 'mercury',
+			host: 'http://muppet.wikia.com',
+			expected: '9412994'
+		}, {
+			vertical: 'gaming',
+			skin: 'mercury',
+			host: 'http://muppet.john.wikia-dev.com',
+			expected: '9412981'
+		}, {
+			vertical: 'entertainment',
+			skin: 'mercury',
+			host: 'http://wowwiki.com',
+			expected: '9412992'
+		}, {
+			vertical: 'lifestyle',
+			skin: 'oasis',
+			host: 'http://muppet.john.wikia-dev.com',
+			expected: '9412973'
+		}, {
+			vertical: 'gaming',
+			skin: 'oasis',
+			host: 'http://muppet.wikia.com',
+			expected: '9412984'
+		}, {
+			vertical: 'entertainment',
+			skin: 'oasis',
+			host: 'http://wowwiki.com',
+			expected: '9412983'
+		}
+	];
+
+	function getModule(vertical, host) {
+		mocks.adLogicZoneParams.getVertical = function() {
+			return vertical;
+		};
+
+		mocks.window.location.host = host;
+
+		return modules['ext.wikia.adEngine.lookup.adapter.appnexusPlacements'](
+			mocks.adLogicZoneParams,
+			mocks.window
+		);
+	}
+
+
+	testCases.forEach(function(testCase) {
+		it('expected placementId is returned for skin/vertical combination ', function() {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Add a semicolon at the end of this statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3ASemicolon)